core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -677,8 +702,9 @@ private[spark] class TaskSetManager(
     }
     if (!successful(index)) {
       tasksSuccessful += 1
-      logInfo("Finished task %s in stage %s (TID %d) in %d ms on %s (%d/%d)".format(
-        info.id, taskSet.id, info.taskId, info.duration, info.host, tasksSuccessful, numTasks))
+      logInfo("Finished task %s in stage %s (TID %d) in %d ms on %s / exec %s (%d/%d)".format(
```

Comment:
maybe "(executor %s)" (favoring parens / verbosity since this is a message that does commonly pop up to users in the console